extensions/amp-render/1.0/storybook/Basic.amp.js

```diff
@@ -1,150 +0,0 @@
-import {withAmp} from '@ampproject/storybook-addon';
```

Comment:
Why did we need to delete the the amp-render storybooks?

Reply:
It is not published to npm, so never have a react binary built. Which means this storybook story will fail to build.

Reply:
Ah gotcha. Shouldn't we keep the amp storybook though? Or does the amp storybooks now use react under-the-hood?

Reply:
amp storybook doesn't actually work.